HISTORY.md

```diff
@@ -16,6 +16,7 @@
 * Fixed a heap use after free in async scan prefetching if dictionary compression is enabled, in which case sync read of the compression dictionary gets mixed with async prefetching
 * Fixed a data race bug of `CompactRange()` under `change_level=true` acts on overlapping range with an ongoing file ingestion for level compaction. This will either result in overlapping file ranges corruption at a certain level caught by `force_consistency_checks=true` or protentially two same keys both with seqno 0 in two different levels (i.e, new data ends up in lower/older level). The latter will be caught by assertion in debug build but go silently and result in read returning wrong result in release build. This fix is general so it also replaced previous fixes to a similar problem for `CompactFiles()` (#4665), general `CompactRange()` and auto compaction (commit 5c64fb6 and 87dfc1d).
 * Fixed a bug in compaction output cutting where small output files were produced due to TTL file cutting states were not being updated (#11075).
+* Fixed a data race on `ColumnFamilyData::flush_reason` caused by concurrent `GetLiveFiles(flush=true)` and other flushes.
```

Comment:
Is this the only case it fixes? flush_reason has been suspicious (in the sense of reporting the wrong reason) many times historically though I don't remember the exact cases

Reply:
Will change this to "...concurrent flushes" instead cuz this is a generic fix.

Reply:
Fixed.